src/Kiota.Builder/Configuration/GenerationConfiguration.cs

```diff
@@ -200,6 +201,11 @@ private string NormalizeDescriptionLocation(string targetDirectory)
         return OpenAPIFilePath;
     }
     public bool IsPluginConfiguration => PluginTypes.Count != 0;
+
+    public bool DisableSSLValidation
```

Comment:
this information should be present in the lock file as well so people running the update command are not locked out.

Reply:
Done